skimage/transform/pyramids.py

```diff
@@ -29,7 +30,7 @@ def _check_factor(factor):
 
 
 def pyramid_reduce(image, downscale=2, sigma=None, order=1,
-                   mode='reflect', cval=0):
+                   mode='reflect', cval=0, multichannel=None):
```

Comment:
For all of these pyramid functions, the message is already there within the call to the `_multichannel_default` routine.